src/Uno.UI/UI/Xaml/Controls/Popup/PopupBase.cs

```diff
@@ -62,11 +66,41 @@ protected override Size ArrangeOverride(Size finalSize)
 			if (newIsOpen)
 			{
 				_openPopupRegistration = VisualTreeHelper.RegisterOpenPopup(this);
+
+				if (IsLightDismissEnabled)
```

Comment:
What happens if IsLightDismissEnabled was false here, but is changed to true after the popup is opened?

Reply:
In such case it will probably be ignored, UWP seems to handle this focus scenario only when `IsOpen` changes